frontend/apps/filemanager/lib/md.lua

```diff
@@ -67,8 +67,8 @@ end
 local lineDelimiters = {'`', '__', '**', '_', '*', '~~'}
 local function findDelim(str, start, max)
     local delim = nil
-    local min = 1/0
-    local finish = 1/0
```

Comment:
I suspect this may be a recommendation for Lua that's completely irrelevant to LuaJIT. (Not an objection, but not support either.)

Reply:
Actually I just noticed which file this is in. Don't touch that file. ;-)

Reply:
Or at least not without bothering upstream about it https://github.com/bakpakin/luamd

```
## Why this is bad
`n / 0` equals `math.huge` when n is positive, and `-math.huge` when n is negative. Use these values directly instead, as using the `/ 0` way is confusing to read and non-idiomatic.
```
But sorry, / 0 confusing to read? I feel a sudden urge to replace all math.huge with 1 / 0, lol

Reply:
As suggested in my opening comment, I also think it's mostly cosmetic.

Reply:
In any case, either put this file or that selene contraption on the ignore list.